packages/build/package.json

```diff
@@ -140,5 +141,10 @@
   },
   "engines": {
     "node": "^12.20.0 || ^14.14.0 || >=16.0.0"
+  },
+  "tsd": {
+    "compilerOptions": {
+      "module": "commonjs"
```

Comment:
why are we using here commonjs? We must expose ESM everywhere we don't use commonjs anymore

Reply:
Plugin fixtures and tsd use commonjs. This doesn't affect the exported types.